backend/entityservice/tests/test_utils.py

```diff
@@ -0,0 +1,64 @@
+import textwrap
+
+import pytest
+
+from entityservice.errors import InvalidConfiguration
+from entityservice.utils import load_yaml_config
+from entityservice.tests.util import generate_bytes, temp_file_containing
+
+
+class TestYamlLoader:
+
+    def test_empty(self):
+        with temp_file_containing(b'') as fp:
+            filename = fp.name
+            assert None == load_yaml_config(filename)
+
+    def test_list(self):
+        with temp_file_containing(b'[1,2,3]') as fp:
+            filename = fp.name
+            assert [1,2,3] == load_yaml_config(filename)
+
+    def test_missing_file(self):
+        filename = 'unlikely a valid file'
+        with pytest.raises(InvalidConfiguration):
+            load_yaml_config(filename)
+
+    def test_random_bytes(self):
+        with temp_file_containing(generate_bytes(128)) as fp:
```

Comment:
there is a non-zero chance that those random bytes resemble valid yaml. Why not just feeding a known invalid config to the test?